src/js/popup.js

```diff
@@ -1466,6 +1469,12 @@ Logic.registerPanel(P_CONTAINER_ASSIGNMENTS, {
           delete assignments[siteKey];
           this.showAssignedContainers(assignments);
         });
+        const resetButton = trElement.querySelector(".reset-button");
+        Utils.addEnterHandler(resetButton, async () => {
+          const pageUrl = `https://${site.hostname}`;
+          const cookieStoreId = Logic.currentCookieStoreId();
+          Utils.resetCookiesForSite(pageUrl, cookieStoreId);
+        });
```

Comment:
enhancement (blocking): Let's add a visual indicator to tell the user that the cookies were deleted.

Reply:
Suggestion: We don't have a component for that in the design system but I think we try to align the style on the message bar component https://acorn.firefox.com/latest/components/message-bar-V5RTUg0H. It could appear at the top of the addon popup when you delete a website data.